{"name": "test_setup_failed_3[21]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <test_setup.TestSetUp object at 0x1272e5a30>.valid", "trace": "self = <test_setup.TestSetUp object at 0x1272e5a30>, a_number = 21\n\n    @pytest.mark.parametrize(\"a_number\", [21, 20])\n    def test_setup_failed_3(self, a_number):\n        \"\"\"\n        test setup failed\n        :return: None\n        \"\"\"\n>       assert self.valid\nE       assert False\nE        +  where False = <test_setup.TestSetUp object at 0x1272e5a30>.valid\n\ntests/tests_setup/test_setup.py:80: AssertionError"}, "description": "\n        test setup failed\n        :return: None\n        ", "parameters": [{"name": "a_number", "value": "21"}], "start": 1722076999062, "stop": 1722076999062, "uuid": "66fa12c4-d96a-4f43-9177-c869383f72f9", "historyId": "066c3f96b92d7668e0bcb16b84a593ca", "testCaseId": "30c10739d2f58af286790dfae347cf47", "fullName": "tests.tests_setup.test_setup.TestSetUp#test_setup_failed_3", "labels": [{"name": "tag", "value": "setup"}, {"name": "tag", "value": "xdist_group(name='setup')"}, {"name": "parentSuite", "value": "tests.tests_setup"}, {"name": "suite", "value": "test_setup"}, {"name": "subSuite", "value": "TestSetUp"}, {"name": "host", "value": "Aymans-MacBook-Pro.local"}, {"name": "thread", "value": "12207-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_setup.test_setup"}]}